### PR TITLE
Create README-FA.md

### PR DESCRIPTION
**Pull request by KooroshMoslemi**

### CODE REVIEW

File: docs/fa/README-FA.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+-->
+
+<!-- Farsi -->
+<!--
+🇫🇦 نسخه فارسی `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) در وبسایت.
+-->
+
+
+<!-- English - Mandarin - Korean - Spanish - Italian - Turkish - Japanese - Arabic - Farsi -->
+[🇬🇧](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) &nbsp; [🇨🇳](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/zh/README-ZH.md) &nbsp; [🇰🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ko/README-KO.md) &nbsp; [🇪🇸](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/es/README-ES.md) &nbsp; [🇮🇹](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/it/README-IT.md) &nbsp; [🇹🇷](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/README-TR.md) &nbsp; [🇯🇵](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ja/README-JA.md) &nbsp; [🇸🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md)&nbsp; [🇫🇦](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/fa/README-FA.md)
```
Comment (Atcold) [inline docs/fa/README-FA.md:43]: Please, add :fr: and let's use :iran: for Persian?

File: docs/fa/README-FA.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+<!-- Arabic -->
+<!--
+🇸🇦 النسخة العربية من ملف `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) والموقع الالكتروني.
+-->
+
+<!-- Farsi -->
+<!--
+🇫🇦 نسخه فارسی `README.md`  [ar](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/ar/README-AR.md) در وبسایت.
+-->
```
Comment (Atcold) [inline docs/fa/README-FA.md:39]:
```suggestion

```

**Review by Atcold — CHANGES_REQUESTED** [review]
Thanks! A few edits.

I think you want to use some RTL tag to make this `MD` look correct.
Also, please update all the other `README-XX.md`s.

File: docs/fa/README-FA.md (hunk 1 above)
Comment (KooroshMoslemi) [reply docs/fa/README-FA.md:8]: OK, added 🇫🇷 , and used 🇮🇷 instead.

**Review by Atcold — APPROVED** [review]
Great! Looks good to me!

**Pull request merged**